css/style.css

```diff
@@ -1 +1,6 @@
-
+html,
+body {
+  font-size: 15px;
+  font-family: 'Montserrat';
+  font-weight: normal; 
```

Comment:
Codacy found an issue: [Expected newline before "}" of a multi-line block (block-closing-brace-newline-before)](https://app.codacy.com/gh/WebHeroSchool/GurevAleksey_itogovaya/pullRequest?prid=6088340)